### DISCUSSION TITLE
What does RSS refers to when you start a Nuxt server?

### DISCUSSION THREAD

**Discussion by ahmadalfy** [nuxt/nuxt#9793] [Questions (Nuxt 2)] [answered]
Upon firing the development server, this message shows up:

```
ℹ Waiting for file changes
ℹ Memory usage: 206 MB (RSS: 316 MB)
ℹ Listening on: http://localhost:3000/
No issues found.
```

What does it refer to? I assume that isn't [Real Simple Syndication](https://en.wikipedia.org/wiki/RSS).

Thank you

**Comment by kissu** [comment]


**Reply by ahmadalfy** [reply]
I am sorry I am not following, what do you mean?

**Comment by danielroe** [comment, marked as answer]
It stands for Resident Set Size (see [docs](https://nodejs.org/api/process.html#process_process_memoryusage_rss)).

> The Resident Set Size, is the amount of space occupied in the main memory device (that is a subset of the total allocated memory) for the process, including all C++ and JavaScript objects and code.

**Reply by ahmadalfy** [reply]
Thank you
